domains/environment_domains/front_door.tf

```diff
@@ -2,16 +2,41 @@ data "azurerm_cdn_frontdoor_profile" "main" {
   name                = var.front_door_name
   resource_group_name = var.resource_group_name
 }
-
 resource "azurerm_cdn_frontdoor_endpoint" "main" {
   for_each = toset(var.domains)
 
-  name                     = each.value
+  name                     = "${each.value}-${local.endpoint_zone_name}"
   cdn_frontdoor_profile_id = data.azurerm_cdn_frontdoor_profile.main.id
   tags                     = var.tags
 }
 
-# TODO: Create origin group
-# TODO: Create origin
+resource "azurerm_cdn_frontdoor_origin_group" "main" {
+  name                     = "${var.environment}-og"
+  cdn_frontdoor_profile_id = data.azurerm_cdn_frontdoor_profile.main.id
+  session_affinity_enabled = true
```

Comment:
Our apps should never rely on session affinity as 12 factor apps are stateless. Can we set it to false?